app/viewModels/routeDetails/transit/AddAnotherOfficeOfTransitViewModel.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewModels.routeDetails.transit
+
+import models.{NormalMode, UserAnswers}
+import play.api.i18n.Messages
+import uk.gov.hmrc.hmrcfrontend.views.viewmodels.addtoalist.ListItem
+import utils.cyaHelpers.routeDetails.TransitCheckYourAnswersHelper
+import javax.inject.Inject
+
+case class AddAnotherOfficeOfTransitViewModel(listItems: Seq[ListItem])
+
+object AddAnotherOfficeOfTransitViewModel {
+
+  def apply(userAnswers: UserAnswers)(implicit messages: Messages): AddAnotherOfficeOfTransitViewModel =
+    new AddAnotherOfficeOfTransitViewModelProvider()(userAnswers)
+
+  class AddAnotherOfficeOfTransitViewModelProvider @Inject() () {
+
+    def apply(userAnswers: UserAnswers)(implicit messages: Messages): AddAnotherOfficeOfTransitViewModel = {
+      val helper = new TransitCheckYourAnswersHelper(userAnswers, NormalMode)
+
+      val listItems = helper.listItems.collect {
+        case Right(value) => value
```

Comment:
We'll also need case Left here for the in progress ones. This was only omitted from the country of routing one as you can't have an in progress thing on a single-page journey